### ISSUE TITLE
Notification of Passing Motions

### ISSUE THREAD

**Issue by mkanwal** [peer/mind#170]
Notification to moderators when a motion passes statistical quorum (so that we don't have to keep checking).

Notification to voters when a discussion/motion is closed and finalized (so they can check the result).

**Comment by mkanwal** [comment]
Email notification to motion and discussion creators with result of motion so that they may start taking action.

**Comment by mitar** [comment]
So all notifications will be sending e-mails. The question is only what notifications we have.

I thought there is already a notification when discussion is closed, no? And motion is closed? You want those?

**Comment by mkanwal** [comment]
Yes, a notification to all voters is sufficient.  We just want to make sure notifications don't become spam, so keeping them personalized is key.

**Comment by mitar** [comment]
So, I am not sure what you are saying now. So please can you be precise which ones you would like added? So your second comment seems like those things already exist? Or?

**Comment by mkanwal** [comment]
I do not have a proposed solution.  I'm saying there are two things to balance: informing members of relevant information while not spamming them.

**Comment by mitar** [comment]
OK, so let's keep this issue about extra in-system notifications which you think are needed. How to then send e-mails to users about those notifications is a separate question.

So which extra notifications would you like to see in-system?
